example/spec/helpers/Collections.js

```diff
@@ -0,0 +1,74 @@
+const { listGranules, removePublishedGranule, deleteGranule } = require('@cumulus/api-client/granules');
+const { getPdrs, deletePdr } = require('@cumulus/api-client/pdrs');
+const { deleteExecution, getExecutions } = require('@cumulus/api-client/executions');
+const { deleteCollection } = require('@cumulus/api-client/collections');
+const { constructCollectionId } = require('@cumulus/message/Collections');
+
+const removeCollectionAndAllDependencies = async (params) => {
```

Comment:
Helper method to proactively remove all sub-dependent records from a collection.

Needs docstring.

Reply:
That's better: 4f7db6588